File: src/GradeClass.java
public class GradeClass {
    private Course course;
    private Grade grade;

    public GradeClass(Course course, Grade grade){
        this.setCourse(course);
        this.setGrade(grade);
    }

    public Course getCourse() {
        return course;
    }

    public Grade getGrade() {
        return grade;
    }

    public void setGrade(Grade grade) {
        this.grade = grade;
    }

    public void setCourse(Course course) {
        this.course = course;
    }
}

File: python_development/GradeClass.py
class GradeClass:
    def __init__(self, course, grade):
        self.course = course
        self.grade = grade
        self.term = None

    def getCourse(self):
        return self.course

    def getGrade(self):
        return self.grade

    def setGrade(self, grade):
        self.grade = grade

    def setCourse(self, course):
        self.course = course

    def setTerm(self, term):
        self.term = term

    def getTerm(self):
        return self.term
